code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Dispenser.dm

```diff
@@ -80,10 +80,47 @@
 	if(istype(L))
 		L.adjust_fire_stacks(amount / 15)
 
-/datum/reagent/ethanol/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+/datum/reagent/ethanol/affect_blood(var/mob/living/carbon/M, var/alien, var/removed) //This used to do just toxin. That's boring. Let's make this FUN.
 	if(issmall(M)) removed *= 2
-	M.adjustToxLoss(removed * 2 * toxicity)
-	return
+	M.nutrition += nutriment_factor * removed
```

Comment:
Let's scrap this line. I think the intention with this being in the ingest proc is to simulate beer and such.

Reply:
Done